PyTAPS/validate_yang.cxx

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright 2019, Akamai Technologies, Inc.
+ * Jake Holland <jholland@akamai.com>
+ * (MIT-licensed, please see LICENSE file in python-asyncio-taps for details)
+ */
+
+/*
+best known reference for libyang:
+https://github.com/CESNET/libyang/blob/master/src/libyang.h.in
+
+git clone https://github.com/CESNET/libyang
+mkdir libyang/build
+cd libyang/build
+cmake -DCMAKE_INSTALL_PREFIX=$HOME/local-installs ..
+make && make install
+cd ../..
+g++ -c -fPIC -I $HOME/local-installs/include validate_yang.cxx -o validate_yang.o
+g++ validate_yang.o -shared -L $HOME/local-installs/lib -lyang -Wl,-soname,libyangcheck.so -o libyangcheck.so
```

Comment:
This only works on Linux/FreeBSD/Solaris.
For MaOS, you need "g++ validate_yang.o -shared -lyang -dynamiclib -o libyangcheck.so" (if you have installed libyang globally)